src/styles/global.css

```
@@ -0,0 +1,34 @@
+@import 'tailwindcss';
+
+@theme {
+  --color-bg: #fdfdfc;
+  --color-body: #1a1a1a;
+  --color-muted: #6b6b6b;
+  --color-border: #e4e4e0;
+  --color-accent: #1d4ed8;
+
+  --font-sans:
+    ui-sans-serif, system-ui, -apple-system, 'Segoe UI', Helvetica, Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Helvetica" to be "helvetica" (value-keyword-case)

(value-keyword-case)